docs/en_us/platform_api/source/courses/blocks.rst

```diff
@@ -0,0 +1,323 @@
+.. _Courses API Blocks Resource:
+
+########################################
+Courses API Blocks Resource
+########################################
+
+With the Courses API **Blocks** resource, you can complete the
+following tasks.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _Get a list of the course blocks in a course:
+
+****************************************
+Get a List of Course Blocks in a Course
+****************************************
+
+The endpoint to get a list of course blocks in a course is
+``/api/courses/v1/blocks/`` with required ``course_id`` parameter.
+
+=====================
+Use Case
+=====================
+
+Get a list of course blocks in a specified course. Response results depend on
+the requesting user's access level.
+
+=====================
+Example Requests
+=====================
+
+GET /api/courses/v1/blocks/?course_id=<course_id>
+
+GET /api/courses/v1/blocks/?course_id=edX%2FDemoX%2FDemo_Course
+&all_blocks=true&requested_fields=graded,format,student_view_multi_device
+
+=====================
+Query Parameters
+=====================
+
+* all_blocks: (boolean) Provide a value of ``true`` to return all blocks.
+
+* block_counts: (list) Specify the types of blocks for which to return a count
+  of that type (including child blocks). Example: ``block_counts=video,problem``.
+
+* block_types_filter: (list) Specify the types of blocks to be included in results.
+  Possible values are: ``sequential``, ``vertical``, ``html``, ``problem``,
```

Comment:
These are only a list of examples.  Possible values are any xBlock type in the system.